apps/api/src/campaign/campaign.controller.ts

```diff
@@ -55,6 +55,18 @@ export class CampaignController {
     return this.campaignService.getUserCampaigns(user.sub)
   }
 
+  // only the user who created the campaign can update it
+  // or the user is in admin role
+  @Get('can-edit/:slug')
+  async canEditCampaign(
+    @Param('slug') slug: string,
+    @AuthenticatedUser() user: KeycloakTokenParsed,
+  ) {
+    const isOwner = await this.campaignService.isUserCampaign(user.sub, slug)
+
+    return isOwner || isAdmin(user)
+  }
+
```

Comment:
I don't think we need an endpoint for that, a [Guard](https://docs.nestjs.com/guards) on the endpoints that need editing permissions will probably do the job.

Reply:
I think he needs it for knowing whether to show the Expense edit button. 
The call to backend could be avoided using the isAdmin function on the frontend which checks the roles of the logged user.

Btw. currently this shows every few seconds if the user is not logged in - so please fix before we merge it:
WARN [HTTP] GET /api/v1/campaign/can-edit/facilis-eveniet-maioresheavily-funded 401 Unauthorized

Reply:
I removed the can-edit endpoint. 
You were right, we can check if the current user is an organizer or admin in the frontend itself with the already available endpoints.